client/src/api/applicationAndTaskSelection.js

```diff
@@ -20,4 +20,13 @@ const getAvailableApplications = async () => {
   return response;
 };
 
-export { getAvailableApplications, fetchTasksFromDB };
+/**
+ * @description Will send a backend call to retrieve all rpa-task objects for the purpose of retrieving the related parameter and possibly output value
+ * @returns {Array} Array of all rpa-task objects
+ */
+const getAllParameters = async () => {
```

Comment:
Not sure about this name, I think we already also renamed the same in the backend, so maybe there could be a better name found for this function

Reply:
I changed it ✅